.github/workflows/CI.yml

```diff
@@ -71,9 +72,11 @@ jobs:
           java-version: 1.8
       - name: Run UATs
         run: |
-          coverage run --source=gdk -m pytest -v -s uat
+          # pytest -v -s uat --gdk-debug
```

Comment:
do we need to keep it here?

Reply:
no we don't. Just wanted to keep it handy, in case someone needs it.